gobblin-service/src/main/java/org/apache/gobblin/service/modules/scheduler/GobblinServiceJobScheduler.java

```diff
@@ -467,6 +467,18 @@ protected static String jobSchedulerTracePrefixBuilder(Properties jobProps) {
         jobProps.getProperty(ConfigurationKeys.FLOW_GROUP_KEY, "<<no flow group>>"));
   }
 
+  /**
+   * Takes a given Date object and converts the timezone to UTC before returning the number of millseconds since epoch
+   * @param date
+   */
+  public static long getMillisecondsSinceEpochInUTC(Date date) {
+    // Create a Calendar object and set it to the given Date
+    Calendar calendar = Calendar.getInstance();
+    calendar.setTime(date);
+    calendar.setTimeZone(TimeZone.getTimeZone("UTC"));
+    return calendar.getTimeInMillis();
+  }
```

Comment:
let's opt for the modern, Joda/`java.time` classes.  I believe something like:
```
ZonedDateTime.of(
    LocalDateTime.ofInstant(date.toInstant(), ZoneId.systemDefault()), 
    ZoneOffset.UTC
).toInstant().toEpochMilli()
```

nit: this is not a accessor (`get`), but a conversion, so `asUTCEpochMillis`